react/features/recording/components/Recording/LocalRecordingManager.ts

```diff
@@ -30,14 +34,31 @@ interface ILocalRecordingManager {
     totalSize: number;
 }
 
+const getMimeType = (): string => {
+    const possibleTypes = [
+        'video/mp4;codecs=h264',
+        'video/webm;codecs=vp9',
+        'video/webm;codecs=vp8',
+        'video/webm;codecs=h264',
+    ];
+    for(let type of possibleTypes) {
+        if(MediaRecorder.isTypeSupported(type)) {
+            return type;
+        }
+    }
+    return 'video/webm;codecs=h264';
```

Comment:
This won't work becasue it's on the above list so it would have succeeded.

Reply:
That would be another scenario in which this is unsupported.

Reply:
Technically the execution will never reach this point but TS complains because the return type of the function is string so there must be a default return

Reply:
throw an exception?